base_layer/core/src/transactions/transaction_components/output_features.rs

```diff
@@ -67,6 +67,11 @@ pub struct OutputFeatures {
     #[serde(default)]
     pub recovery_byte: u8,
     pub metadata: Vec<u8>,
+    // TODO: Add these fields
+    // pub contract_id: Option<FixedHash>,
+    // pub constitution: Option<ContractConstitution>,
```

Comment:
Do we want to add flags for these as well?
Or are we going with the presence of them is enough?

Reply:
OutputFlags already exist for the constitution. Whenever a contract_id is present we likely want one of those flags to be set